class UserNotFoundError(Exception):
   pass

class HabitNotFoundError(Exception):
   pass
